src/python/WMCore/MicroService/DataStructs/MSRuleCleanerWflow.py

```diff
@@ -54,7 +182,15 @@ def docSchema(self):
             'ParentageResolved': Bool,
             'PlineMarkers': None,
             'IsClean': False
-            'ForceArchive', False]
+            'IsLogDBClean': False,
+            'IsArchivalDelayExpired': False,
+            'ForceArchive': False,
+            'RequestTransition': [],
+            'IncludeParents': False
+            'DataPileup': [],
+            'MCPileup': [],
+            'InputDataset': 'None',
```

Comment:
Did you mean `None` instead of `'None'`?

Reply:
Yes, indeed.